packages/@webex/plugin-cc/src/metrics/MetricsManager.ts

```diff
@@ -270,4 +275,46 @@ export default class MetricsManager {
   public static resetInstance() {
     MetricsManager.instance = undefined;
   }
+
+  public static getCommonTrackingFieldForAQMResponse(response: any): Record<string, any> {
+    const fields = {
+      agentId: response?.data?.agentId || EMPTY_STRING,
+      teamId: response?.data?.teamId || EMPTY_STRING, // TODO: handle multiple teams
```

Comment:
Is there a Jira to address this? If today there are multiple teams, would this break somehow?